test/e2e/osa_test.go

```diff
@@ -83,4 +84,43 @@ var _ = Describe("Openshift on Azure e2e tests", func() {
 		})
 		Expect(err).ToNot(HaveOccurred())
 	})
+
+	It("should run the correct image", func() {
+		// image format is either:
+		// docker.io/openshift || registry.access.redhat.com/openshift3
+		format := strings.Split(c.cs.Config.Images.Format, "/")
+		prefix := fmt.Sprintf("%v/%v", format[0], format[1])
+
+		// these pods should have the same prefix
+		podsToCheck := map[string]struct{}{
+			"oauth-proxy":                    {},
+			"origin-control-plane":           {},
```

Comment:
The suffix for RHEL is `ose-control-plane`. Similar I guess about all other origin instances.

Reply:
Thanks for pointing this out.  I am currently reworking the test and will attempt to determine ose vs origin.